Samples.Swift/Samples.Swift/ViewControllers/AddItemViewConroller/AddItemViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  AddItemViewController.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/9/21.
+//  Copyright © 2021 PayPal. All rights reserved.
+//
+
+import PayPalCheckout
+import UIKit
+
+protocol AddItemViewControllerDelegate {
+  func didTapSaveWithItem(item: PurchaseUnit.Item, at index: Int)
+}
+
+class AddItemViewController: UIViewController {
+
+  var delegate: AddItemViewControllerDelegate?
+  var index: Int?
+
+  let popupBox: UIView = {
+    let view = UIView()
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.backgroundColor = .white
+    view.layer.cornerRadius = 8
+    return view
+  }()
```

Comment:
is there a reason for this to not be lazily instantiated? for consistency's sake, practically does not matter :)

Reply:
No reason per-se, will update to be consistent!